src/index.js

```diff
@@ -9,7 +12,17 @@ import runAudit from './lib/run-audit/index.js';
 
 dotenv.config();
 
+const puppeteerCacheDir = join(homedir(), '.cache', 'puppeteer');
+
+export const onPreBuild = async ({ utils } = {}) => {
+  // Puppeteer relies on a global cache since v19.x, which otherwise would not be persisted in Netlify builds
+  await utils?.cache.restore(puppeteerCacheDir);
```

Comment:
Yeah, I think it would be good to add something here and when saving. The Deploy Log now adds a section for `onPreBuild`, but without any content it may prompt questions.

<img width="860" alt="image" src="https://user-images.githubusercontent.com/720908/214262859-0503b0c1-fa29-4393-8fb7-46c4896f479e.png">

Reply:
Yeah agreed, looks a bit weird empty like that. I've added a couple of lines:

![](https://user-images.githubusercontent.com/20773163/214268728-72f3900c-f25a-4ec6-ab26-dcf29a28dd96.png)